physionet-django/events/models.py

```diff
@@ -153,3 +153,49 @@ def withdraw(self, comment_to_applicant=None):
             comment_to_applicant (str): The comment to add to the participant about the status.
         """
         self._apply_decision(self.EventApplicationStatus.WITHDRAWN, comment_to_applicant)
+
+
+class EventDataset(models.Model):
+    """
+    Captures information about datasets for events.
+    """
+    class EventDatasetStatus(models.TextChoices):
+        Active = 'AC', _('Active')
```

Comment:
Thinking about it more, maybe we should remove the Active/Inactive status? We don't actually provide a mechanism of activating/deactivating event datasets, and the functionality seems duplicative of removing an event dataset.

Reply:
For sure, did you mean to remove this class and use status as a simple boolean? @alistairewj

Reply:
But yes, this is used to set status as inactive once someone removes the dataset from event, and we never let them reactivate it, so this looks unnecessary(unless we want the option to allow them to reactivate a dataset, but that sounds like an extra feature/work and maybe its just better with a simple boolean)

Reply:
So, perhaps the best resolution is, why don't you make a commit where you remove all references to the `EventDatasetStatus`, and we can keep the commit in the history. If we decide to add the feature in later, it will be easy to bring it back. Then I think we are good to merge.

Reply:
Sounds good. doing that right now